README.rst

```diff
@@ -42,6 +42,23 @@ To install this extension, use the following command::
 
    pip install pyota[ccurl]
 
+Optional Local Pow
+==================
+To perform Proof-of-Work locally without relying on any node,
+you can install an extension module called `PyOTA-PoW`_ .
+
+Specifiy the ``local_pow=True`` argument when creating an
+api instance, that will redirect all ``attach_to_tangle``
+API calls to an interface function in the ``pow`` package.
+Returns the same as a node would for the API call.
+
+To install this extension, use the following command::
+
+   pip install pyota[pow]
+
+Alternativley you can take a look on the repository
+`Ccurl.interface.py`_  to install Pyota-PoW.
+Follow the steps depicted in the repo's README file.
```

Comment:
❔ Out of curiosity, could we also use `Ccurl.interface.py` as a replacement for `pyota-ccurl`?

Reply:
Theoretically I think it is possible, but requires a bit more work on `Ccurl.interface.py`. I'm not sure about how this would work on Windows. Btw, does PyOTA supports Windows at all?

Reply:
🤔 Good question; I don't think I ever explicitly added Windows support, but I don't think there's anything OS-specific in that code, so it probably should work.  At any rate, I never got any complaints from Windows users... well [except for one](https://github.com/todofixthis/pyota-ccurl/issues/22), but that turned out to be `pip`'s fault 😹